src/lib/serviceRoutinesV2/putEntityAttributeValue.cpp

```diff
@@ -79,6 +81,15 @@ std::string putEntityAttributeValue
     }
   }
 
+  if (ciP->httpStatusCode == SccConflict)
+  {
+    ErrorCode   ec("TooManyResults", "There is more than one entity that match the update. Please refine your query.");
```

Comment:
Desperate intent of persuasion ... :-):
- What if the matching is on a pattern?
- What if the matching is per id **and type** of the entity?

"that id" would no longer be 100% valid.
"More than one matching entity" covers all the possibilities, apart from being shorter and more "elegant" ...